src/index.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Enzyme from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import axios from 'axios';
+// import PhotoGrid from './components/PhotoGrid.js';
+//import Photos from './components/Photos.js';
+
+Enzyme.configure({ adapter: new Adapter() });
+
+console.log('testing');
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      photos: []
+    }
+  }
+
+  componentDidMount() {
```

Comment:
Good to know about this react method! I used old fashioned $.ajax to GET from server and then passed the data to the App as props